_drafts/2015-07-06-web-audio-article-1.md

```diff
@@ -0,0 +1,106 @@
+
+###Before Web Audio API there was the HTML audio element
+
+The introduction of the `audio` element in HTML5 was important because it allowed us to have audio playback across different browsers without having to run plugins such as Flash or Quicktime. Previously, if one does not have those plugins installed, the audio or video content in the web page cannot be properly downloaded or displayed. This was one of the common issues in regards to audio and video as the required plugins may be missing, outdated, blocked or in conflict with another plugin or extension.
+
+However even with the `audio` element, there were still huge limitations on using audio with sophisticated games or interactive applications such as: 
+
+1. **No precise timing controls**
+
+	To build various musical applications, it's very important to have consistent and precise timing of audio events. Not just having sounds to start and stop but to be able to schedule complex scenarios such as playing musical sequences or rhythms. This is now possible with Web Audio API. However, working the audio clock is still one of the least understood topics as it can be quite complex. A good explanation of it by Chris Wilson can be found [here](http://www.html5rocks.com/en/tutorials/audio/scheduling/).
+
+2. **Limited number of sounds can played at once**
+
+	It only allows single sound layering (polyphonic), a single audio stream can only be played once at a time.
+
+3. **No reliable way to pre-buffer a sound**
+	
+	Audio streams can't be loaded unless triggered by a user touch event (`onmousedown`, `onmouseup`, `onclick`, or `ontouchstart`) and this is futher limited in Safari mobile browsers which does not support the `preload` attribute and will fail when an audio stream is played on page load.
+
+4. **No way to apply real-time effects**
+
+5. **No way to analyse sounds**
+
+6. **Not all browsers support the same audio file format**
+
+	This has been a really big issue that often made developers [angry](http://phoboslab.org/log/2011/03/the-state-of-html5-audio). 
+
+In all there are workarounds for some of these issues however with the introduction of Web Audio API, it has solved most these problems and made audio on the web easier for developers to use. It's not all perfect though, as there are still [issues](https://github.com/WebAudio/web-audio-api/issues) being discovered.
+	
```

Comment:
Maybe a small section which talks about 1. The [current status](http://webaudio.github.io/web-audio-api/) of the WebAudio spec. And 2. The [browser support](http://caniuse.com/#search=audio-api) for WebAudio here would be relevant??